packages/design-system-docs/src/scripts/components/PageHeader.jsx

```diff
@@ -8,7 +8,7 @@ class PageHeader extends React.PureComponent {
       const classes = classNames('ds-c-badge ds-u-margin-left--1 ds-u-text-transform--capitalize', {
         'ds-c-badge--success': this.props.status === 'Ready',
         'ds-c-badge--warn': this.props.status === 'Draft',
-        'ds-c-badge--error': this.props.status === 'Deprecated',
+        'ds-c-badge--error': this.props.status === 'Do not use',
```

Comment:
Use `ds-c-badge--alert` for the red background badge: 
```suggestion
        'ds-c-badge--alert': this.props.status === 'Do not use',
```